### PR TITLE
Python investigation renames

### PR DESCRIPTION
**Pull request by jonasnick**

### CODE REVIEW

File: python/example.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ async def participant(
         state2, eq_round1 = participant_step2(hostseckey, state1, cmsg1)
     except UnknownFaultyParticipantOrCoordinatorError as e:
         if blame_mode:
```
Comment (real-or-random) [inline python/example.py:103]: `investigation_mode`?

File: python/example.py [hunk 2]
```diff
@@ -129,7 +129,7 @@ async def coordinator(
     # If the coordinator implements blame mode and it is enabled, it sends an
     # extra message to the participants.
     if blame_mode:
-        blame_msgs = coordinator_blame(pmsgs1)
+        blame_msgs = coordinator_investigate(pmsgs1)
```
Comment (real-or-random) [inline python/example.py:132]:
```suggestion
        inv_msgs = coordinator_investigate(pmsgs1)
```

File: python/tests.py [hunk 3]
```diff
@@ -249,10 +249,10 @@ def simulate_chilldkg(
 def simulate_chilldkg_full(
     hostseckeys,
     t,
-    blame: bool,
+    investigation: bool,
 ) -> List[Optional[Tuple[chilldkg.DKGOutput, chilldkg.RecoveryData]]]:
     # blaming is not supported by this wrapper
```
Comment (real-or-random) [inline python/tests.py:254]:
```suggestion
    # Investigating is not supported by this wrapper
```

File: python/chilldkg_ref/encpedpop.py [hunk 4]
```diff
@@ -242,22 +242,22 @@ def participant_step2(
...
-def participant_blame(
-    blame_state: ParticipantBlameState,
-    cblame: CoordinatorBlameMsg,
+def participant_investigate(
+    error: UnknownFaultyParticipantOrCoordinatorError,
+    cinvestigation: CoordinatorInvestigationMsg,
 ) -> NoReturn:
-    simpl_blame_state, enc_secshare, pads = blame_state
-    enc_partial_secshares, partial_pubshares = cblame
+    simpl_blame_state, enc_secshare, pads = error.blame_state
```
Comment (real-or-random) [inline python/chilldkg_ref/encpedpop.py:249]:
```suggestion
    simpl_invstate, enc_secshare, pads = error.invstate
```

File: python/chilldkg_ref/chilldkg.py [hunk 5]
```diff
@@ -600,14 +589,14 @@ def participant_finalize(
     return dkg_output, RecoveryData(eq_input + cmsg2.cert)
 
 
-def participant_blame(
-    blame_state: ParticipantBlameState,
-    cblame: CoordinatorBlameMsg,
+def participant_investigate(
+    error: UnknownFaultyParticipantOrCoordinatorError,
+    cinvestigation: CoordinatorInvestigationMsg,
```
Comment (real-or-random) [inline python/chilldkg_ref/chilldkg.py:594]: I think we can use a short form "inv" in *variable names* (everywhere):
```suggestion
    cinv: CoordinatorInvestigationMsg,
```

**Review by real-or-random — COMMENTED** [review]
Concept ACK, that's nice

File: python/example.py (hunk 1 above)
Comment (jonasnick) [reply python/example.py]: Fixed

File: python/example.py (hunk 2 above)
Comment (jonasnick) [reply python/example.py]: Fixed

File: python/chilldkg_ref/chilldkg.py (hunk 5 above)
Comment (jonasnick) [reply python/chilldkg_ref/chilldkg.py]: Fixed

**Comment by jonasnick** [comment]
First commit should be ready for review now. Still working on the second commit.

**Comment by jonasnick** [comment]
Ready for review now

**Review by real-or-random — APPROVED** [review]
ACK

**Pull request merged**